### PR TITLE
Add "Why Rhino?" and clean up main readme

### LINKED ISSUE #179: Documentation: Why Rhino? How is it different from other frameworks?

### PR DESCRIPTION
**Pull request by marekrogala**
### Changes
Closes #179 

### How to test
Build documentation, see if all links work and all text roughly makes sense.
If something needs addressing, please fix it if possible before merging to speed things up (I'm OOO tomorrow).

After merging, ping @ianemoore to let him know this needs his 👀  :)

### CODE REVIEW

**Review by TymekDev — CHANGES_REQUESTED** [review]
I have not read through vignettes just yet, but dropped a bunch of small suggestions for the README.

File: .Rbuildignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+^renv$
+^renv\.lock$
```
[the remark this answers is not in the document]
Comment (kamilzyla) [reply .Rbuildignore:2]: We don't need it, but doesn't really help either.

[the remark this answers is not in the document]
Comment (kamilzyla) [reply .Rbuildignore]: Done.

File: README.md [hunk 2]
```diff
@@ -11,123 +9,39 @@ _**Disclaimer: Rhino is under an active development. Before version 1.0.0 releas
...
-  * [:recycle: Migrate an Existing Shiny Application to Rhino](#recycle-migrate-an-existing-shiny-application-to-rhino)
-  * [:wrench: Configure Rhino with `rhino.yml`](#wrench-configure-rhino-with-rhinoyml)
-    * [`rhino.yml` Taxonomy](#rhinoyml-taxonomy)
-  * [:construction: Develop a Shiny Application with Rhino](#construction-develop-a-shiny-application-with-rhino)
-* [Contributing](#contributing)
-* [About](#about)
+# Why Rhino?
+Rhino allows you to create Shiny apps "The Appsilon Way"  - like a fullstack software engineer: apply best software engineering practices, modularize your code, test it well, make UI beautiful and think about adoption from the very beginning. It is an opinionated framework with a focus on software engineering practices and development tools.
+
+Rhino supports your work in 3 main areas:
```
[the remark this answers is not in the document]
Comment (kamilzyla) [reply README.md:15]: Done.

File: README.md [hunk 3]
```diff
@@ -11,123 +9,39 @@ _**Disclaimer: Rhino is under an active development. Before version 1.0.0 releas
...
+
+1. Clear code (scalable app architecture, modularization based on Box and Shiny modules.
+2. Quality (unit tests, E2E tests with Cypress, logging and monitoring, linting).
+3. Automation (project startup, CI with Github Actions, dependency management with renv, configuration management with config, Sass and Javascript bundling with ES6 support via Nodejs).
+
+These features are often implemented using well-known packages. The biggest value in Rhino is that you instantly get all of them working together and pre-configured.
```
[the remark this answers is not in the document]
Comment (kamilzyla) [reply README.md:21]: Done.

File: README.md [hunk 4]
```diff
@@ -11,123 +9,39 @@ _**Disclaimer: Rhino is under an active development. Before version 1.0.0 releas
...
-## Contributing
-Pull requests are welcome! Please see our [contributing guidelines](.github/CONTRIBUTING.md) for more details.
-
-| Tool           | Command                  | `devtools` equivalent    | Comment
-|----------------|--------------------------|--------------------------|-
-| Unit tests     | `testthat::test_local()` | `devtools::test()`       |
-| Linter         | `lintr::lint_package()`  | `devtools::lint()`       |
-| `pkgdown` site | `pkgdown::build_site()`  | `devtools::build_site()` | If built successfully, the website will be in `docs` directory. Requires `pkgdown` version >= 2.0.0.
 
+- 🏗️: `rhino::init()` creates a new Shiny Application with Rhino.
```
[the remark this answers is not in the document]
Comment (kamilzyla) [reply README.md:38]: Standardized. I decided to remove icons for now - they looked bad on the pkgdown site in my browser.

**Review by kamilzyla — APPROVED** [review]
LGTM. Applied suggestions from @TymekDev and adjusted the structure (move "About" to "Explanation" as agreed with @jakubnowicki).

**Review by TymekDev — APPROVED** [review]

File: vignettes/explanation-what-is-rhino.Rmd [hunk 5]
```diff
@@ -31,4 +31,4 @@ You may want to use Rhino if:
 
 # Why not use Golem, Leprechaun?
 Golem is a great package, and we believe it is a good fit for many developers. Choose whichever works best for you and your project!
-You can learn more about the differences in [How is Rhino different from ...?](articles/about-how-is-it-different.html). 
\ No newline at end of file
+You can learn more about the differences in [How is Rhino different from ...?](articles/about-how-is-it-different.html).
```
Comment (jakubnowicki) [inline vignettes/explanation-what-is-rhino.Rmd:34]:
```suggestion
You can learn more about the differences in [Similar projects](articles/explanation-similar-projects.html).
```

**Review by jakubnowicki — APPROVED** [review]
Looks good, two comments.

**Pull request merged**